js/views/userPageVw.js

```diff
@@ -1444,6 +1467,72 @@ module.exports = baseVw.extend({
 
   },
 
+  addModeratorClick: function(e){
+    var $targ = $(e.target).closest('.js-addmoderator');
```

Comment:
Could you please consolidate your variable declarations at the top? I know that we're not super consistent in our style, but in regards to this we mostly are. For example:

```javascript
var var1 = 'hello',
    var2 = 'bye',
    varWithoutInitialVal;
```